### PR TITLE
Experimental features

### PR DESCRIPTION
**Pull request by marekrjpolak**
## Description

Add support for experimental features.
Záznam-obrazovky-z-2024-06-26-16-45-11

### CODE REVIEW

**Comment by MiroslavProchazka** [comment]
Looks good! Let's sync with product next week and finalize the design etc. Good job!

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        padding-top: ${spacingsPx.md};
+        margin-top: ${spacingsPx.md};
+        border-top: 1px solid ${({ theme }) => theme.borderElevation2};
+    }
+`;
+
+const FeatureLine = ({
+    feature,
+    features,
+}: {
```
Comment (peter-sanderson) [inline packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: nit: I would prefer extract type

```ts
type FeatureLineProps = {....}

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    return (
+        <FeatureLineWrapper>
+            <TextColumn
+                title={titleId ? <Translation id={titleId} /> : feature}
+                description={descId && <Translation id={descId} />}
+            />
+            <ActionColumn>
+                <Checkbox
+                    isChecked={checked}
+                    onClick={() =>
```
Comment (peter-sanderson) [inline packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: nit: I would prefer when its multiline to extraxct it to funcion

```ts
const handleOnClick = () => { 
...
```

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 1 above)
Comment (marekrjpolak) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Less lines => why not.

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 2 above)
Comment (marekrjpolak) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Less lines => why not.

File: packages/suite/src/constants/suite/experimental.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+export enum ExperimentalFeature {
+    FullRbf = 'full-rbf',
+    NoShitcoins = 'no-shitcoins',
+}
+
+type FeatureIntlMap = Partial<Record<ExperimentalFeature, TranslationKey>>;
+
+export const ExperimentalFeatureTitle: FeatureIntlMap = {
+    [ExperimentalFeature.FullRbf]: 'TR_EXPERIMENTAL_FULL_RBF',
+    [ExperimentalFeature.NoShitcoins]: 'TR_EXPERIMENTAL_NO_SHITCOINS',
```
Comment (tomasklim) [inline packages/suite/src/constants/suite/experimental.ts]:
```suggestion
    [ExperimentalFeature.NoBitcoin]: 'TR_EXPERIMENTAL_NO_BITCOIN',
```

**Review by tomasklim — DISMISSED** [review]

File: packages/suite/src/constants/suite/experimental.ts (hunk 3 above)
Comment (marekrjpolak) [reply packages/suite/src/constants/suite/experimental.ts]: abc

**Comment by marekrjpolak** [comment]
Early access:
Snímek obrazovky z 2024-07-03 11-40-21
Experimental features:
Snímek obrazovky z 2024-07-03 11-40-36
Early access AND experimental features:
Snímek obrazovky z 2024-07-03 11-40-16

Current **Experimental features** settings section in desktop Suite:
image

**Review by hynek-jina — APPROVED** [review]
Nice
- We can improve copy afterwards since it is behind the debug
- Let's add first real feature soon 💪

**Comment by MiroslavProchazka** [comment]
Copy adjustments please: 

Experimental features
For experienced users only. Use at your own risk.
Select which experimental features to enable in Trezor Suite to experience the latest advancements and innovations. Note, these features are in testing, may be unstable, and might not have long-term support.

**Comment by MiroslavProchazka** [comment]
Trezor Password Manager
Trezor Password Manager is a free, open-source browser extension that allows you to use your Trezor device in place of a master password.

**Comment by MiroslavProchazka** [comment]
Trezor Password Manager
Trezor Password Manager is an extension within Trezor Suite that allows you to use your Trezor device in place of a master password.

**Pull request merged**